### PR TITLE
Update InverseFinance.json

### PR DESCRIPTION
**Pull request by xiaona423**
Inverse - zh

### CODE REVIEW

**Review by DarkGhost7 — APPROVED** [review]

**Comment by DarkGhost7** [comment]
@Bee926 can you tl check?

**Comment by Bee926** [comment]
will do, by the way, do these need to be done in separate PRs?

**Comment by DarkGhost7** [comment]
> will do, by the way, do these need to be done in separate PRs?

No they do not, but she did them in a separate pr so I reviewed them

File: data/protocols/1/InverseFinance.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     },
     "zh": {
       "name": "Inverse Finance",
-      "description": "Inverse Finance is a suite of permissionless decentralized finance tools governed by Inverse DAO. Their main products are Anchor: a money market, DOLA: stablecoin, and Dollar Cost Averaging Vaults. Users can have a say in Inverse DAO with its native INV token."
+      "description": "Inverse Finance 是一套无需许可的去中心化金融工具，由 Inverse DAO 管理。主要产品有，Anchor: 金融市场，DOLA: 稳定币，以及“美元平均成本”机枪池。Inverse DAO 是由 INV 的持有者来管理的。"
```
Comment (Bee926) [inline data/protocols/1/InverseFinance.json:40]: change 美元平均成本
to 定期定额投资法

**Pull request merged**